Persephone/MPDClient/Models/MPDSong.swift

```diff
@@ -52,6 +54,13 @@ extension MPDClient {
       return getTag(.date)
     }
 
+    var path: String {
+      return uriString
+        .split(separator: "/")
+        .dropLast()
+        .joined(separator: "/")
```

Comment:
There might be an `NSString` helper function/property to do this -- I've used the opposite (`NSString.lastPathComponent`), so maybe there's an inverse?

Reply:
Looks like there is; `deletingLastPathComponent`, https://developer.apple.com/documentation/foundation/nsstring/1411141-deletinglastpathcomponent